src/app/components/mnoe-api/admin/teams.svc.coffee

```diff
@@ -0,0 +1,22 @@
+@App.service 'MnoeTeams', ($q, MnoeAdminApiSvc, toastr, MnoErrorsHandler) ->
+
+  teamsApi = (id) ->
+    MnoeAdminApiSvc.one('organizations', id).all('teams')
+
+  @list = (organization_id,limit, offset, sort) ->
+    params = {}
+    params['order_by'] = sort
+    params['limit'] = limit
+    params['offset'] = offset
```

Comment:
1. Not sure what the point of using the `object['key']` syntax here? The key names are not dynamic, nor special characters.
2. Not sure if defining these properties in separate lines is useful? The values being set are not conditional (`params['order_by'] = sort if blah`). Consider defining it in one line `params = { order_by: sort, limit: limit, offset: offset }`

Reply:
it was just readability, pointless readability to be fair. cheers